odoo/tools/config.py

```diff
@@ -397,8 +398,8 @@ def die(cond, msg):
         # Ensures no illegitimate argument is silently discarded (avoids insidious "hyphen to dash" problem)
         die(args, "unrecognized parameters: '%s'" % " ".join(args))
 
-        die(bool(opt.syslog) and bool(opt.logfile),
-            "the syslog and logfile options are exclusive")
+        die(bool(opt.syslog) and bool(opt.logfile) and bool(opt.logfilesdb),
+            "the syslog, logfile and logfilesdb options are exclusive")
```

Comment:
Conditional is wrong, it fails only when you set all 3 options but they are mutually exclusive with one another
```suggestion
        die(bool(opt.syslog) + bool(opt.logfile) + bool(opt.logfilesdb) > 1,
            "the syslog, logfile and logfilesdb options are exclusive")
```